server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -80,7 +80,10 @@ export default class DashboardPresenter {
   private readonly showAssignedCaseworkerColumn =
     this.dashboardType === 'All open cases' || this.dashboardType === 'Completed cases'
 
-  readonly isSearchable = this.dashboardType === 'All open cases' || this.dashboardType === 'Unassigned cases'
+  readonly isSearchable =
+    this.dashboardType === 'All open cases' ||
+    this.dashboardType === 'Unassigned cases' ||
+    this.dashboardType === 'Completed cases'
```

Comment:
To be clear @sldblog are you looking for a dashboard enum such as `Dashboard.ALL_OPEN_CASES` instead?

Reply:
@sldblog Suggesting we merge in with string based comparisons at the moment, and work up a patch which will replace with and enum, need to modify a number of files to do so - hence don't want to delay sprint goal.

Reply:
@awhodgson256 I wasn't clear, sorry. We should change `dashboardType` to an enum in a new PR, yes.

However, the `const property = this.dashboardType === A || this.dashboardType === B` pattern encourages typos, errors, and shotgun surgery: to understand the impact of one dashboard change, we need to read all properties.

It would be great if that could be fixed inline with extending the branching on this property